packages/parser-typescript-config/tests/typescript-config.ts

```diff
@@ -0,0 +1,124 @@
+import * as sinon from 'sinon';
+import test from 'ava';
+import { EventEmitter2 } from 'eventemitter2';
+
+import TypeScriptConfigParser from '../src/typescript-config';
+
+test.beforeEach((t) => {
+    t.context.sonarwhal = new EventEmitter2({
+        delimiter: '::',
+        maxListeners: 0,
+        wildcard: true
+    });
+});
+
+test.serial('If any file is parsed, it should emit a `notfound::typescript-config` error', async (t) => {
```

Comment:
I think this tests can run in parallel

Reply:
I'm not so sure, we are using `spy` in that tests, so it can be problematic to run in parallel

Reply:
but we are spying on a variable that is created in the context of each individual test (`new EventEmitter2()`), we are not reusing any global variable so it should be fine.

Reply:
you are right!